packages/itwinui-css/src/tabs/default.scss

```diff
@@ -78,7 +77,7 @@
 
   &.iui-large {
     .iui-tab {
-      height: $iui-component-height-large;
+      min-height: $iui-component-height-large;
```

Comment:
How would you feel about setting a `max-height` or a `-webkit-line-clamp` to prevent extreme cases like this?
![Screen Shot 2022-06-09 at 12 46 21 PM](https://user-images.githubusercontent.com/849817/172900668-0f444901-c810-464e-b64d-cfa0ee58fa21.png)

Reply:
I dont know... What would you feel you like to see here? Limit height as a standard?

Reply:
I'd be fine with setting it to 3.  If a user needs more lines it wouldn't be hard for them to override the default.

Reply:
It is 0.2.0
![image](https://user-images.githubusercontent.com/81580355/175255987-0f3b7a54-7d42-4ea2-88ad-3722f1d3cd42.png)

Sure too bad?
Excluding full refactor, I would need to add new class on label `iui-tab-name` and then move this whole selector outside of tabs mixin.

Reply:
If you look at how the user would normally override this, they need to apply a custom classname on the Tab and then target our `iui-tab-label` like so:

```scss
.my-tab > .iui-tab-label {
  // ...
}
```

Because our specificity is 0-2-0, this will not always work (depending on order of imports) and then they might reach for `!important`. So yeah, it's better to reduce it to 0-1-0.

Why do you need a new class? You could use the same `iui-tab-label`.

Reply:
what? I need to apply overflow rules on both, name and description divs. I cannot apply it just on parent, it would not work.
So I need to introduce a new class for the first div, because currently it doesnt have any.

Reply:
The `> *` selector can help with that, no need for a new class.

Reply:
ok, moved into mixin, now it has 0.1.0